nano/node/nodeconfig.cpp

```diff
@@ -448,6 +459,47 @@ nano::error nano::node_config::deserialize_json (bool & upgraded_a, nano::jsonco
 	return json.get_error ();
 }
 
+std::string nano::node_config::serialize_frontiers_confirmation (nano::frontiers_confirmation_mode mode_a) const
+{
+	switch (mode_a)
+	{
+		case nano::frontiers_confirmation_mode::always:
+			return "always";
+		case nano::frontiers_confirmation_mode::automatic:
+			return "auto";
+		case nano::frontiers_confirmation_mode::lazy:
+			return "lazy";
+		case nano::frontiers_confirmation_mode::disabled:
+			return "disabled";
+		default:
+			return "auto";
+	}
+}
+
+nano::frontiers_confirmation_mode nano::node_config::deserialize_frontiers_confirmation (std::string const & string_a)
+{
+	if (string_a == "always")
+	{
+		return nano::frontiers_confirmation_mode::always;
+	}
+	else if (string_a == "auto")
+	{
+		return nano::frontiers_confirmation_mode::automatic;
+	}
+	else if (string_a == "lazy")
+	{
+		return nano::frontiers_confirmation_mode::lazy;
+	}
+	if (string_a == "disabled")
```

Comment:
Was an else-if meant here? Not that it matters functionally in this case.

Reply:
Thanks